lib/ecto/uuid.ex

```diff
@@ -183,24 +204,49 @@ defmodule Ecto.UUID do
     end
   end
 
+  @default_version 4
+  @default_options version: 4
   @doc """
-  Generates a random, version 4 UUID.
+  Generates a uuid with the given options.
   """
   @spec generate() :: t
-  def generate(), do: encode(bingenerate())
+  @spec generate(options) :: t
+  def generate(opts \\ [@default_options]), do: encode(bingenerate(opts))
```

Comment:
probably don't need the `@default_options` and could just use an empty list since you supply a default version with `Keyword.get/3`